sky/cli.py

```diff
@@ -231,6 +231,22 @@ def _parse_env_var(env_var: str) -> Tuple[str, str]:
     return ret[0], ret[1]
 
 
+def _parse_secret_var(secret_var: str) -> Tuple[str, str]:
+    """Parse secret vars into a (KEY, VAL) pair."""
+    if '=' not in secret_var:
```

Comment:
I'm wondering a corner case where the secret var contains `=` (e.g. a based encoded token) and that might lead to unexpected behavior. We need a unit test for this

Reply:
Oh, it should be fine since it will only appears in values, which must come after the `=` of `KEY=VAL`